src/renderer.ts

```diff
@@ -43,6 +43,39 @@ export class Renderer {
 		});
 	}
 
+	public async renderSignature(document: vscode.TextDocument, signatureHelp: vscode.SignatureHelp): Promise<string> {
+
+		if (signatureHelp.signatures.length === 0) {
+			return '';
+		}
+
+		let parts: any[] = [];
+		const activeSignature: any[] = [];
+
+		signatureHelp.signatures.forEach((signatureInformation: vscode.SignatureInformation, index: number) => {
+			let pushTo;
+			if (signatureHelp.activeSignature === index) {
+				pushTo = activeSignature;
+			} else {
+				pushTo = parts;
+			}
+			pushTo.push(this.getMarkdown(
+				`${String(index + 1).padStart(2, '0')}/${String(signatureHelp.signatures.length).padStart(2, '0')} ${signatureHelp.activeSignature === index ? '🟩' : '⬛'} \`\`\`${signatureInformation.label}\`\`\``));
```

Comment:
Instead of hardcoding these colors, can you use one of the VS Code's theme colors [to style](https://code.visualstudio.com/api/extension-guides/webview#theming-webview-content) the active signature (such as `editorHoverWidget.highlightForeground`)

Reply:
Also not convinced the number of signatures is useful in this case. We show this count in the parameter hints widget because we only show one signature at a time, but since we are showing all the signature I suggest omitting counts

Reply:
I was using emojis for the two squares. I will make use of `editorHoverWidget.highlightForeground` somehow. Need to figure out how to do it..

I included the index number aligned to let users correlate it with the number they see parameter hints widget. But I am OK to remove it.

Reply:
I am thinking of eliminating the squares and the issue of colors is moot.

Reply:
I still am keeping the matching signature at the top of the view.